### PR TITLE
Allow modders to make longer range Mass Drivers

### PR DESCRIPTION
**Pull request by MEEPofFaith**
Increased lifetime and made the bullet just oof if it passes over an oofed target.

Compiling is down, so no prior testing has been made.

### CODE REVIEW

**Comment by Anuken** [comment]
Even with longer range, mass driver bullets still slow down in transit.

**Comment by Syncinus** [comment]
I'm pretty sure the ``b.remove`` method calls despawned anyway, so you don't have to called despawned and then remove.

File: core/src/mindustry/entities/bullet/MassDriverBolt.java [hunk 1]
```diff
@@ -47,14 +47,17 @@ public void update(Bulletc b){
...
-        //if the target is dead, just keep flying until the bullet explodes
-        if(data.to.dead()){
-            return;
-        }
-
         float baseDst = data.from.dst(data.to);
         float dst1 = b.dst(data.from);
         float dst2 = b.dst(data.to);
+        
+        //if the target is dead, oof upon passing the destination point
```
Comment (Anuken) [inline core/src/mindustry/entities/bullet/MassDriverBolt.java:54]: Why should the bullet just disappear when it flies past a mass driver that isn't there anymore? There's nothing to stop it.

**Comment by MEEPofFaith** [comment]
Ima fix my PR

**Pull request closed without merging**